src/hooks/usePerfBrowser.ts

```diff
@@ -306,7 +306,7 @@ const usePerfBrowser = (
                 const point = serie.points.find((p) => p[xAxisMode] === rank);
 
                 let perf = '-';
-                if (point?.perf) {
+                if (point?.perf !== undefined && point?.perf !== null) {
```

Comment:
Total JS noob here and this is a nitpick but wouldn't it be more appropriate to check that `typeof point?.perf === "number"` if we're going to run `toFixed` on it ?

Reply:
In pure JS that would be true but as it is Typescript it knows that point.perf is a number if it isn't undefined or null, because it is typed : `number | undefined | null`. However you are right that making the condition on `typeof point?.perf === "number"` would give the same result for less code and be more logical to read. I'll make the change thanks 👍